GitCommands/Git/Executable.cs

```diff
@@ -220,6 +221,27 @@ public StreamReader StandardError
                 }
             }
 
+            /// <inheritdoc />
+            public async Task TerminateAsync(bool entireProcessTree = true, bool tryCtrlC = true, int exitTimeoutMilliseconds = IProcess.DefaultExitTimeoutMilliseconds)
```

Comment:
Is there a scenario where `entireProcessTree = false`?

Reply:
> Is there a scenario where `entireProcessTree = false`?

I think we should call it with `false` in order to not kill e.g. a diff tool with unsaved user changes by mistake.
Since the other value is not used, I can remove the argument in the next fixup commit.